src/createServer.js

```diff
@@ -1,9 +1,42 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+const url = require('url');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+    const parts = normalizedUrl.pathname.slice(1).split('/');
```

Comment:
you need to filter empty strings and you omitted "test" from this failed test:
![image](https://github.com/user-attachments/assets/abeba22b-4dd2-4424-a6cc-424c3c547e75)

Reply:
this test still fails because the normalizedUrl.pathname for this test omits 'test' from req.url. Try creating normalizedUrl differently